### PR TITLE
Adding new property to teams app definition

### PR DESCRIPTION
**Pull request by subray2014**

### CODE REVIEW

File: api-reference/beta/resources/teamsAppInstallationScope.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Types of [teamsAppInstallationScope](./teamsAppInstallationScope.md).
+
+## Members
+
+| Member | Value| Description |
+|:---------------|:--------|:----------|
```
Comment (AkJo) [inline api-reference/beta/resources/teamsAppInstallationScope.md:21]: We do not have files per enum.

You could add an entry in the centralized enum resource file or the place where the enum is being referenced.

File: api-reference/beta/resources/teamsappdefinition.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ The details of a version of a [teamsApp](teamsapp.md).
 | azureADAppId        | string   | The WebApplicationInfo.id from the Teams App manifest. |
 | displayName         | string   | The name of the app provided by the app developer. |
 | version             | string   | The version number of the application. |
+| allowedInstallationScopes | [teamsAppInstallationScope](./teamsAppInstallationScope.md) collection | A collection of scopes where 1) the Teams app can be installed in and 2) for which data regarding the scopes can be accessed by the app. |
```
Comment (AkJo) [inline api-reference/beta/resources/teamsappdefinition.md:28]: Needs some wordsmithing.

Also, please list out the values like you did for publishingState.

File: api-reference/beta/resources/enums.md [hunk 3]
```diff
@@ -2148,3 +2148,10 @@ Possible values for user account types (group membership), per Windows definitio
 |other|
 |unknownFutureValue|
 
+### teamsAppInstallationScope values
+
+|Member    |Value    |Description |
+|:---------|:--------|:----------- |
+|team      |0        |Indicates the Teams app may be installed within a team and data about the team is authorized to be accessed by the app.|
```
Comment (AkJo) [inline api-reference/beta/resources/enums.md:2155]: Indicates that the Teams app may be installed within a team and is authorized to access that team's data.

File: api-reference/beta/resources/enums.md [hunk 4]
```diff
@@ -2148,3 +2148,10 @@ Possible values for user account types (group membership), per Windows definitio
...
+|groupChat |1        |Indicates the Teams app may be installed within a group chat and data about the group chat is authorized to be accessed by the app.|
```
Comment (AkJo) [inline api-reference/beta/resources/enums.md:2156]: Indicates that the Teams app may be installed within a group chat and is authorized to access that group chat's data.

File: api-reference/beta/resources/enums.md [hunk 5]
```diff
@@ -2148,3 +2148,10 @@ Possible values for user account types (group membership), per Windows definitio
...
+|personal  |2        |Indicates the Teams app may be installed for the user personally and data about the user is authorized to be accessed by the app.|
```
Comment (AkJo) [inline api-reference/beta/resources/enums.md:2157]: Indicates that the Teams app can be installed in the personal scope of a user and is authorized to access that user's data.

File: api-reference/beta/resources/teamsappdefinition.md [hunk 6]
```diff
@@ -25,6 +25,7 @@ The details of a version of a [teamsApp](teamsapp.md).
 | azureADAppId        | string   | The WebApplicationInfo.id from the Teams App manifest. |
 | displayName         | string   | The name of the app provided by the app developer. |
 | version             | string   | The version number of the application. |
+| allowedInstallationScopes | [teamsAppInstallationScope](./enums.md) collection | A collection of scopes where the Teams app can be installed in. Possible values are:</br>`team` — Indicates the Teams app may be installed within a team and data about the team is authorized to be accessed by the app. </br>`groupChat`  — Indicates the Teams app may be installed within a group chat and data about the group chat is authorized to be accessed by the app. </br> `personal` — Indicates the Teams app may be installed for a user personally and data about the user is authorized to be accessed by the app. |
```
Comment (AkJo) [inline api-reference/beta/resources/teamsappdefinition.md:28]: Please update the descriptions based on the comments above.

File: api-reference/beta/resources/teamsappdefinition.md [hunk 7]
```diff
@@ -25,6 +25,7 @@ The details of a version of a [teamsApp](teamsapp.md).
 | azureADAppId        | string   | The WebApplicationInfo.id from the Teams App manifest. |
 | displayName         | string   | The name of the app provided by the app developer. |
 | version             | string   | The version number of the application. |
```
Comment (AkJo) [inline api-reference/beta/resources/teamsappdefinition.md:27]: Also, please update this file appcatalogs-list-teamsapps.md with an example:

List the basic details of only those apps that can be installed in the personal scope of a user.
GET appCatalogs/teamsApps?$expand=appDefinitions($select=id,displayName,allowedInstallationScopes)&$filter=appDefinitions/any(a:a/allowedInstallationScopes has 'personal')

File: api-reference/beta/api/appcatalogs-list-teamsapps.md [hunk 8]
```diff
@@ -452,6 +452,59 @@ Content-Type: application/json
...
+
+{
+    "@odata.context": "https://canary.graph.microsoft.com/testprodbetateamsgraphdev/$metadata#appCatalogs/teamsApps(appDefinitions(id,displayName,allowedInstallationScopes))",
+    "value": [
+        {
+            "id": "5a542e1c-5f8c-4793-8b0c-6082464b2378",
+            "externalId": "4b3ec336-b998-4623-9e25-d4182fb82159",
+            "displayName": "Carriage",
+            "distributionMethod": "organization",
+            "appDefinitions@odata.context": "https://canary.graph.microsoft.com/testprodbetateamsgraphdev/$metadata#appCatalogs/teamsApps('5a542e1c-5f8c-4793-8b0c-6082464b2378')/appDefinitions(id,displayName,allowedInstallationScopes)",
```
Comment (AkJo) [inline api-reference/beta/api/appcatalogs-list-teamsapps.md:495]: beta url

**Review by AkJo — APPROVED** [review]
Please fix the issues called out.

File: api-reference/beta/resources/teamsappdefinition.md [hunk 9]
```diff
@@ -25,6 +25,7 @@ The details of a version of a [teamsApp](teamsapp.md).
...
+| allowedInstallationScopes | [teamsAppInstallationScope](./enums.md) collection | A collection of scopes where the Teams app can be installed in. Possible values are:</br>`team` — Indicates that the Teams app can be installed within a team and is authorized to access that team's data. </br>`groupChat`  — Indicates that the Teams app can be installed within a group chat and is authorized to access that group chat's data. </br> `personal` — Indicates that the Teams app can be installed in the personal scope of a user and is authorized to access that user's data. |
```
Comment (Lauragra) [inline api-reference/beta/resources/teamsappdefinition.md:28]: Remove the link to the enums.md file; just list the name in the type column and define the values in the description, as you have done.

Comment (subray2014) [reply api-reference/beta/resources/teamsappdefinition.md]: Removed the link to enums.md

**Review by Lauragra — APPROVED** [review]

**Pull request merged**